comment/__init__.py

```diff
@@ -1 +1,8 @@
 default_app_config = 'comment.apps.CommentConfig'
+
+__version__ = "2.0.0"
+
+
+def _get_version():
```

Comment:
Why do you use `_` prefix in the function name since you are using it outside the module?

Reply:
I have read in Python(also in Django) `_` is also used to show an internal function(or class or attribute or simply an object) which might be changed in the future without any notice.

Reply:
I am not sure about that but what I know is that in python the underscore before the function or the variable means that this member is private and shouldn't be used outside the scope.

Reply:
To quote the official `django` documentation:

> **APIs marked as internal**
   Certain APIs are explicitly marked as “internal” in a couple of ways:
    Some documentation refers to internals and mentions them as such. If the documentation says that something is internal, we reserve the right to change it.
    Functions, methods, and other objects prefixed by a leading underscore (_). This is the standard Python way of indicating that something is private; if any method starts with a single _, it’s an internal API.


Reference: https://docs.djangoproject.com/en/3.1/misc/api-stability/#apis-marked-as-internal